webapp/app/forms/flows/multistep_flow.py

```diff
@@ -91,8 +94,10 @@ def handle(self, step_name):
         override_session_data(stored_data)
 
         if render_info.redirect_url:
+            logger.info(f"Redirect to {render_info.redirect_url}", exc_info=True)
```

Comment:
Why `exc_info=True`? That adds exception information, but AFAICT we can't be in an exception context here?

Reply:
You are totally right and this was a copy paste mistake